v20.1/show-backup.md

```diff
@@ -21,7 +21,14 @@ Only members of the `admin` role can run `SHOW BACKUP`. By default, the `root` u
 Parameter | Description
 ----------|------------
 `location` | The location of the backup to inspect. For more details, see [Backup File URLs](backup.html#backup-file-urls).
-`WITH privileges` | <span class="version-tag">New in v20.1:</span> List which users and roles had which privileges on each database and table in the backup.
+`kv_option_list` | Control the show behavior with a comma-separated list of [these options](#options).
+
+### Options
+
+Option       | Value | Description
+-------------+-------+-----------------------------------------------------
+`privileges` | N/A   | <span class="version-tag">New in v20.1:</span> List which users and roles had which privileges on each database and table in the backup.
```

Comment:
I believe this is just table privileges. We are talking about how to better display database level metadata but I'm fairly sure we don't have it today.